src/common/utils/model.ts

```diff
@@ -81,9 +80,10 @@ export function fetchAllStateProjections(snapshotUrl: string | null = null) {
       APIRegionSubPath.STATES,
     );
     return all
-      .filter(
-        summaryWithTimeseries =>
-          getStateName(summaryWithTimeseries.state || '') !== undefined,
+      .filter(summaryWithTimeseries =>
+        summaryWithTimeseries?.state
+          ? Boolean(regions.findByStateCode(summaryWithTimeseries.state))
```

Comment:
Minor (feel free to ignore if you just want to merge as-is), but using `Boolean()` as a way to check for null seems a bit confusing / indirect.

I'd probably use a `!== null` check and/or create an `isKnownStateCode()` helper.

Reply:
Ah, thanks for the callout.  I'm accustomed to prefer `Boolean` to indicate we care in a boolean context whether the value is truthy or not, and avoid issues of `null` vs `undefined`, etc.  If we were using `strictNullChecks`, maybe it would matter less to me.  Or, if I hadn't been burned by `undefined`s failing a `!== null` check too many times to count : )

I'll merge as is, but agree that an `isKnownStateCode` would be a worthy improvement.